docs/configuring.md

```diff
@@ -27,8 +29,40 @@ Please note that you will have to recompile the application for the changes to t
 
 
 ## Google Analytics
+
 If you wish to track pages hits, set up [Google analytics](https://analytics.google.com/analytics/web/#embed/report-home/a76615855w155408876p157084577/) for your domain, then add your google analytics id to your config.edn files (mentioned above) or environment variables. This is completely optional.
 
 ```clojure
   {:google-analytics "UA-12345678-9"}
-``
+```
+
+
+# Example configuration
+
+All keys can be overridden with environment variables, such as `DATABASE_URL=http://...`
+These properties are read in by [yogthos](https://github.com/yogthos/), which can read in variables in a variety of ways
+See [Yogthos's config docs](https://github.com/yogthos/config#yogthosconfig) for more details.
+
+Copy and paste the code below and save it as `config.edn`, in the same `config` subfolder you inted to use. Don't forget to tweak it to match your preferred properties!
+
+```clojure
```

Comment:
Maybe link to the template rather than replicating it? Otherwise they could get out of sync :)